elements/pfe-primary-detail/README.md

```diff
@@ -56,8 +56,30 @@ This component is an implementation of one of the "Primary detail simple list in
 ```
 
 ### Accessibility
+
 The default markup should have semantic markup if the component can't load, once it loads the component the appropriate tab interactions and appropriate markup for assistive tech is handled for you.
 
+#### Focus Indicator Styles
+
+The component requires visible focus indicator styles for focusable elements (ie.`links`, `buttons`, `[tabindex="0"]`) in order to meet [**WCAG 2.0/2.1 AA compliance**](https://www.w3.org/WAI/WCAG21/quickref/#focus-visible). Below is a good example of styles to use for the focus indicator, these styles match the focus indicator of `pfe-navigation`. The `padding` style is to increase the clickable area of links in order to help users have a better experience when trying to click the links, this also helps users with limited mobility.
```

Comment:
Though great information, since this is now being managed by the component, I don't think we need to ask developers to add any styles anymore.  I'd like to avoid requiring developers to write any custom styles in general so that we maintain a low-effort for implementation.

Reply:
This is for the slotted content, we only address the nav part of this component, not what's in the pane. I think it's good to point out that just using our components doesn't get a site an A+ on a11y.

Reply:
That seems like great global advice that is not specific to this component then; would you like to find a good place to add it [here](https://github.com/patternfly/patternfly-elements/pull/1421)?

Reply:
It also sounds like a place where pfe-base could offer assistance - is there anything missing in pfe-base styles to meet standards?  If so, I'd like to get it included because ideally, you can get everything you need from the project and you have the _option_ to write custom styles if you want to.

Reply:
That sounds great! I think having a reminder that pane styles need to be accessible and a link to centralized docs would be :100:

@kelsS if you want to take a crack at pfe-base I'd like to do it as a separate ticket/PR/effort

Reply:
@kelsS If you would like, I have a PR I just opened to add in the broadcasted variable hook to pfe-base.scss.  If you find an update we can make to our base class (basically our version of normalize.css but a lot less opinionated), you're welcome to include it in this PR: https://github.com/patternfly/patternfly-elements/pull/1530/files

Reply:
The readme for pfe-styles is a great place to store notes on making your light DOM styles accessible and we can migrate these notes to the main documentation site after the migration.